tests/test_vectorstores_milvus.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Please update the scripts name into `test_vectorstores_langchain_milvus.sh`, so the test could be triggered when the related folder changes.